js/ui/overlay/ui.overlay.js

```diff
@@ -1416,7 +1405,7 @@ const Overlay = Widget.inherit({
         this._updateZIndexStackPosition(false);
         this._toggleTabTerminator(false);
 
-        this._actions = null;
+        this._actions = this._scrollHandler = this._$prevTargetParents = null;
```

Comment:
it's better to use `undefined` to not show this cache variables in instance object